### DISCUSSION TITLE
Lines plugin broken

### DISCUSSION THREAD

**Discussion by ezeholz** [lowlighter/metrics#457] [Q&A] [answered]
Hello! Using the Lines plugin, I discover that If the user has a private repo that's not being skipped, the plugin throws an error like this:
![image](https://user-images.githubusercontent.com/30635862/128645742-c416a257-baa8-4b1b-ae47-b6ed4f491c6c.png)
Instead of changing the action to skip every private repo every single time it's created, isn't better to just handle it inside the plugin itself? Is there a way of doing that?
Cheers! :D

**Comment by lowlighter** [comment, marked as answer]
Fetched repositories depends on the scope on your token, so if the `base` plugin is able to get them, they should be able to be processed by the `lines` plugin. 

Since it seems to affect private repositories it seems a scope issue, though it may not for the reason above.

I'll dig into it later, but yes indeed this an issue

**Reply by ezeholz** [reply]
For you to know, the repo token scopes are the following:
![image](https://user-images.githubusercontent.com/30635862/128771956-0f6dc1e0-e403-4139-9ae8-dea94d0eddc9.png)
Also, I have this option activated in general:
![image](https://user-images.githubusercontent.com/30635862/128773848-8cd1e648-1417-48ae-9838-db1992ec103f.png)

**Reply by lowlighter** [reply]
This should have been fixed in cb741f0c1 👍

**Reply by ezeholz** [reply]
I can confirm, It's working as expected. Thanks! :D
